nat-lab/tests/test_hole_punch.py

```diff
@@ -22,20 +22,23 @@ async def test_hole_punch() -> None:
         PUBLIC_IP = "10.0.254.5"
         LOCAL_PORT = 1000
         PUBLIC_PORT = 1001
-        process = connection_1.create_process([
-            "upnpc",
-            "-i",
-            "-a",
-            "192.168.105.88",
-            str(LOCAL_PORT),
-            str(PUBLIC_PORT),
-            "udp",
-        ])
+        process = connection_1.create_process(
+            [
+                "upnpc",
+                "-i",
+                "-a",
+                "192.168.105.88",
+                str(LOCAL_PORT),
+                str(PUBLIC_PORT),
+                "udp",
+            ]
+        )
```

Comment:
Is this reformatting needed? It makes it harder to tell if there is any change here

Reply:
I think I might have had the wrong version of black installed. I installed the correct version of black and reformatted. There should not be any more pure formatting changes